libfwupdplugin/fu-security-attrs.c

```diff
@@ -157,12 +157,17 @@ fu_security_attrs_remove_all(FuSecurityAttrs *self)
  *
  * Calculates the HSI string from the appended attributes.
  *
+ * If hsi_uint_ret is NULL, the guint type hsi will not be returned.
```

Comment:
I think you want instead (a few lines further up) `@hsi_number: (out) (nullable): the HSI number`

Reply:
Thanks for correcting this. I'll revise it.